docs/plugins.rst

```diff
@@ -38,14 +38,16 @@ the ``PLUGIN_PATHS`` list can be absolute or relative to the settings file::
 Where to find plugins
 =====================
 
-We maintain a separate repository of plugins for people to share and use.
-Please visit the `pelican-plugins`_ repository for a list of available plugins.
+We maintain a separate GitHub organization for plugins so contributors have
+a highly visible location and an easier way to collaborate with fellow users.
+Please visit the `getpelican-plugins`_ organization to access these plugins.
 
-.. _pelican-plugins: https://github.com/getpelican/pelican-plugins
+.. _pelican-plugins: https://github.com/getpelican-plugins
```

Comment:
I think this target name needs to be changed to `getpelican-plugins` (that's what the Travis build reports at least).